### DISCUSSION TITLE
Live update / refresh pulls

### DISCUSSION THREAD

**Discussion by Grubnedlo** [TedJuch/TedTournament#15] [General]
Awesome extension! I can't seem to get the function to update "live" data points without re-applying (removing =, adding back). My sheet is shared, could other active viewers be an issue here? Using scores and game status.

**Comment by TedJuch** [comment]
This is an issue with how custom functions work with sheets. You might try going to **File > Settings > Calculation > On change and every minute.**

Sometimes this helps. Otherwise, you might need to consider not using TedTournament directly and use something like `=importrange("1DyuuT9zPSh9RdzrAF_1bY6HhyuYKckL3E6wr-sGKZTs","Men 2023 Tournament Data!A:V")`.

This will pull in the dataset that TedTournament uses and you can just do vlookups by game number. This should update in more real-time.

**Comment by jessestein** [comment]
Is the spreadsheet updating?  I don't seem to be getting any results for Today's game  (2023, second round, day 1 (03-18-2023)  Checked the source sheet... looks like the team names are there, but no other data.

**Reply by TedJuch** [reply]
I'm tracking some issues with my source. It should repair itself once the source issues are resolved.

**Reply by jessestein** [reply]
thanks, working again, probably has for some time, but just wanted to confirm

**Comment by TedJuch** [comment]
Great, thanks!
